charts/loghouse/values.yaml

```diff
@@ -107,9 +110,10 @@ clickhouse:
 # if you install images from private registry
   imagePullSecrets:
 # Our chart convert memory limit to clickhouse max memory limit
-# It wants 256Mi for system use. So, specify 512Mi limit or more
+# It wants 120Mi for system use. So, specify 2Gi limit or more
```

Comment:
```suggestion
# Note that "max memory limit" is calculated as "limits.memory" - 128Mi,
# because clickhouse process needs 128Mi to run.
# So specify at least 2Gi for "limits.memory".
```